Source/Core/Core/HW/EXI/BBA/BuiltIn.cpp

```diff
@@ -289,6 +218,78 @@ StackRef* CEXIETHERNET::BuiltInBBAInterface::GetTCPSlot(u16 src_port, u16 dst_po
   return nullptr;
 }
 
+std::optional<std::vector<u8>>
+CEXIETHERNET::BuiltInBBAInterface::TryGetDataFromSocket(StackRef* ref)
```

Comment:
I know this was just a move but can this function be `const`?

Reply:
The use of `ResolveAddress` in the next commit following the move will prevent me from doing that. A new UDP peer will have its IP address added to the ARP table with a new MAC address generated.

Reply:
Whoops, glossed over that they were the same function.  Yeah, that makes sense.  Thanks